package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "graphiql-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint ./src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "npm run lint -- --fix",
+    "preview": "vite preview",
+    "prettier": "npx prettier src --check",
+    "prettier:fix": "npm run prettier -- --write",
+    "prepare": "husky install",
+    "test": "vitest",
+    "coverage": "vitest run --coverage.enabled --coverage.all"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "npx vitest --run"
+    }
+  },
+  "lint-staged": {
+    "**/*.{ts,tsx}": [
+      "npm run lint:fix",
+      "npm run prettier:fix"
+    ]
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^2.0.1",
+    "@testing-library/react-hooks": "file:latest",
+    "firebase": "^10.7.1",
```

Comment:
you don't need `@testing-library/react-hook` in dependencies. Move in devDependencies